bolt/util.py

```diff
@@ -1,11 +1,16 @@
+import os
```

Comment:
- swap out `os` for `pathlib` and just use below
- split imports into built-in, third-party, and package / local
- sort imports alphabetical